tests/test_integration_25_1_24_2.py

```diff
@@ -0,0 +1,621 @@
+# Copyright (C) 2024 - 2025 ANSYS, Inc. and/or its affiliates.
```

Comment:
I decided to copy this entire module, because so many of the test cases changed between 25 R1 and the current development version.